ccd-definition/CaseEvent/User/UserEventsCUI.json

```diff
@@ -7,6 +7,7 @@
     "DisplayOrder": 6,
```

Comment:
rename file to UserEventsCUI-nonprod.json

Reply:
File is renamed to CUI-nonprod.json

Reply:
From the other PR, it is found that if renamed to CUI-nonprod.json, there will be error comes in smoketest:aat (Unknown CaseField 'applicant1Represented' for CaseType 'CIVIL' in worksheet 'AuthorisationCaseField'), so decide to keep the same name. Related PR: https://github.com/hmcts/civil-ccd-definition/pull/1904/commits